### DISCUSSION TITLE
Changing Sunsynk settings daily - longevity

### DISCUSSION THREAD

**Discussion by StefanB7** [kellerza/sunsynk#142] [General]
Hi. Thanks so much for this project, it is super nice!

I am currently changing my Inverter's load limit settings each night to 'Essentials', thereby ensuring my batteries aren't used to power non-essentials and each morning I switch it back to 'Zero Export'. This has been working nicely (using a 3ph inverter).
However, I am concerned about the longevity of the inverter should I change the settings each day. Is it something I should be concerned about? They probably use flash to store the settings, say the flash can withstand 10 000 write cycles, at this rate I'll reach that in 365*2 = 730, 10 000 / 730 = 13 years.
Is this something I should be concerned about?
Thanks in advance.

**Comment by kellerza** [comment]
@StefanB7 it probably depends on what type of flash they use, the SLC ones can do 100k cycles. The inverter should be storing energy stats (daily/monthly/yearly), very likely on the same storage, so they hopefully used something a bit more robust!

I've not heard of anyone's inverter failing due to this, but might be worth searching/asking on powerforum or energytalk

**Comment by kellerza** [comment]
btw, I also did this for a while, but after seeing #140 I'm now trying to just limiting the output from the batteries to 800W during the early morning when I switch on the geyser.

**Comment by StefanB7** [comment]
Thanks for your response @kellerza.
I reached out to Sunsynk and they said it shouldn't be a problem:
![image](https://github.com/kellerza/sunsynk/assets/22056509/11d5528c-5804-4a56-ad89-80a2abc8f088)
